src/patternfly/components/Alert/alert.scss

```diff
@@ -3,215 +3,174 @@
   --pf-c-alert--BoxShadow: var(--pf-global--BoxShadow--lg);
   --pf-c-alert--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-alert--grid-template-columns: max-content 1fr max-content;
-  --pf-c-alert--grid-template-rows: 1fr auto;
+  --pf-c-alert--grid-template-rows: var(--pf-global--spacer--lg) auto auto;
+  --pf-c-alert--before--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-alert--before--BorderTopColor: var(--pf-global--default-color--200);
+  --pf-c-alert--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-alert--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-alert--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-alert--PaddingLeft: var(--pf-global--spacer--md);
 
   // Icon
-  --pf-c-alert__icon--Padding: var(--pf-global--spacer--md);
-  --pf-c-alert__icon--Color: var(--pf-global--default-color--300);
-  --pf-c-alert__icon--BackgroundColor: var(--pf-global--default-color--100);
-  --pf-c-alert__icon--FontSize: var(--pf-global--icon--FontSize--lg);
+  --pf-c-alert__icon--Color: var(--pf-global--default-color--200);
+  --pf-c-alert__icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-alert__icon--MarginTop: calc((var(--pf-global--spacer--xs) / 2));
+  --pf-c-alert__icon--FontSize: var(--pf-global--icon--FontSize--md);
 
   // Title
   --pf-c-alert__title--FontSize: var(--pf-global--FontSize--md);
   --pf-c-alert__title--Color: var(--pf-global--default-color--300);
-  --pf-c-alert__title--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-alert__title--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-alert__title--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-alert__title--PaddingLeft: var(--pf-global--spacer--md);
 
-  // Description
-  --pf-c-alert__description--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-alert__description--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-alert__description--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-alert__description--MarginTop: calc(-1 * var(--pf-global--spacer--sm));
+  // Alert description
+  --pf-c-alert__description--PaddingTop: var(--pf-global--spacer--xs);
 
   // Action
-  --pf-c-alert__action--PaddingTop: #{pf-size-prem(11px)}; // one off spacer to align properly.
-  --pf-c-alert__action--PaddingRight: var(--pf-global--spacer--xs);
+  --pf-c-alert__action--MarginTop: #{pf-size-prem(-6px)};
```

Comment:
You can use `var(--pf-global--spacer--form-element)` or `var(--pf-c-button--PaddingTop)` here.